ChangeLog

```diff
@@ -22,6 +22,9 @@ Release date: TBA
 ..
   Put bug fixes that should not wait for a new minor version here
 
+* Fixed a false positive for ``unused-import`` where everything
+  was not analysed properly inside typing guards
```

Comment:
```suggestion
  was not analyzed properly inside typing guards.
```
I usually try to end changelog entries with a full stop.

Reply:
👍

Reply:
I copied the entry from #5397 to avoid a conflict for @Pierre-Sassoulas but he will have to update his PR